packages/stencil/components/pagination/src/components/osds-pagination/styles/osds-pagination.size.scss

```diff
@@ -0,0 +1,21 @@
+@import '~@ovhcloud/ods-theming/size/ods-component-size';
+@import './osds-pagination.mixins';
+
+@mixin osds-pagination-theme-size() {
+  osds-button::part(button) {
+    width: 2.25rem;
+    height: 2.25rem;
+    border-radius: var(--ods-size-border-radius-01);  
+    padding: 0;
+  }
+
+
+  .arrows {
+    padding-left: 0.5rem;
```

Comment:
here too.
avoid using hard coded rem.
see @astagnol for explain with css variables